vaadin/info.yaml

```
@@ -0,0 +1,4 @@
+url: https://github.com/vaadin/quarkus
+issues:
+  repo: quarkusio/quarkus
+  latestCommit: 32985
```

To build the 2.x branch, you need to set a new job (in the same [workflow](https://github.com/vaadin/quarkus/blob/master/.github/workflows/quarkus-snapshot.yaml)) and specify the environment variable `ALTERNATIVE="2.x"` for example 

```suggestion
  latestCommit: 32985
alternatives:
    2.x:
      issue: <ISSUE_NUMBER_IN_YOUR_REPO>
      quarkusBranch: "2.16"
```